docs/Administrator Guide/GlusterFS Introduction.md

```diff
@@ -21,7 +21,7 @@ Gluster is a scalable, distributed file system that aggregates disk storage reso
 
 
 Enterprises can scale capacity, performance, and availability on demand, with no vendor lock-in, across on-premise, public cloud, and hybrid environments.
-Gluster is used in production at thousands of enterprises spanning media, healthcare, government, education, web 2.0, and financial services.
+Gluster is used in production at thousands of enterprises, spanning media, healthcare, government, education, web 2.0, and financial services.
```

Comment:
I do not think the added comma is correct, it might be better to replace "enterprises" with "organisations" and leave the comma out.